docs/SDKs/Terra-js/Common-examples.md

```diff
@@ -0,0 +1,78 @@
+# Common examples
+
+## Configuring LCDClient
+
+```ts
+import { LCDClient } from "@terra-money/terra.js";
+
+const lcd = new LCDClient({
+  URL: "https://bombay-lcd.terra.dev/",
+  chainID: "bombay-12",
+  gasPrices: [new Coin("uluna", "0.01133")],
+  gasAdjustment: "1.5",
+  gas: 10000000,
+});
+```
+
+The rest of the examples assume LCDClient was initialized similar to the above example. 
```

Comment:
I moved this phrase to the previous section. If you want to keep it here instead, let's say, "The following examples assume you initialized the LCDClient by using the previous code example or similar code.